lib/bot.js

```diff
@@ -310,6 +322,14 @@ class Bot {
     }
   }
 
+  getDiscordAvatar(nick) {
+    const user = this.discord.users.find('username', nick);
+    if (user) {
+      return user.avatarURL;
+    }
+    return null;
+  }
+
```

Comment:
So that if you have two users with the same nickname barring case, you can pick a particular one to use. (It's also possible to have two users with the exact same nickname, case-sensitively, but then the ambiguity is basically required as opposed to mostly quite avoidable.)

Maybe it's over-much, though – if two users have the same nickname, case-sensitive or not, it's not particularly unreasonable to expect weird behavior around the ambiguity.